guevaraTorres/assignments/fastSlow/lc876/lc876.js

```diff
@@ -0,0 +1,89 @@
+// Problem Statement #
```

Comment:
Nice pseudocode and great to see you're typing the solution twice to improve muscle memory!